pbrEngine/guiStateDistinguisher.py

```diff
@@ -99,19 +99,49 @@ def distinguishStart(self, data):
 
 # start menu
 _map_start = {
-    "Wii Remote Control Sideways": PbrGuis.START_WIIMOTE_INFO,
+    "Wii Remote Control Sideways"                  : PbrGuis.START_WIIMOTE_INFO,
+    "Configuración horizontal del mando de Wii"    : PbrGuis.START_WIIMOTE_INFO,  # es
+    "Type de commandes horizontal"                 : PbrGuis.START_WIIMOTE_INFO,  # fr
+    "Steuerung C"                                  : PbrGuis.START_WIIMOTE_INFO,  # de
+    "Telecomando Wii - Orizzontale"                : PbrGuis.START_WIIMOTE_INFO,  # it
+    "Ｗｉｉリモコン「よこ」"                           : PbrGuis.START_WIIMOTE_INFO,  # ja
 
     # This is the only one currently needed
-    "Choose a Game Mode"                 : PbrGuis.START_MENU,
-    "Selección de modo de juego"         : PbrGuis.START_MENU,  # es
-    "Choisir un mode de jeu"             : PbrGuis.START_MENU,  # fr
-    "Spielmodus wählen"                  : PbrGuis.START_MENU,  # de
-    "Scegli una modalità di gioco"       : PbrGuis.START_MENU,  # it
-    "ゲームモードを　えらんでください"         : PbrGuis.START_MENU,  # ja
-
-    "Choose Options"                     : PbrGuis.START_OPTIONS,
-    "Save the changed options settings?" : PbrGuis.START_OPTIONS_SAVE,
-    "Announcer's Voice"                  : PbrGuis.START_VOICE,
-    "Colosseum Mode"                     : PbrGuis.START_MODE,
-    "Continue"                           : PbrGuis.START_SAVEFILE, # doesn't work, but relying on unstucker anyway
+    "Choose a Game Mode"                           : PbrGuis.START_MENU,
+    "Selección de modo de juego"                   : PbrGuis.START_MENU,  # es
```

Comment:
can you please figure out a way of formatting that avoids diffs on otherwise unchanged lines?

Reply:
the spacing is changed to line up better with longer lines, if that's not wanted I can revert it and only have the spacing change on the longer ones

Reply:
ooooor, figure out a way of formatting that avoids diffs on otherwise unchanged lines. for example, a newline